### ISSUE TITLE
The Qt Deploy Tool doesn't take the given arguments

### ISSUE THREAD

**Issue by ItsJustMiaouss** [qtproject/qt-labs-vstools#20]
**Describe the bug**
The Qt Deploy Tool (windeployqt) arguments appear to be ignored after the build when setting them on the project properties page.

Note: I ensured that I was using the correct configuration profile.

Deploy configuration:

![image](https://github.com/qt-labs/vstools/assets/59478524/db1b9f7b-0a15-40ed-8f95-2578490ae433)

Build folder:

![image](https://github.com/qt-labs/vstools/assets/59478524/563d6c0c-de37-4dc3-9951-e3501c2fc4a6)

Console output:
```
Build started...
1>------ Build started: Project: TestApp, Configuration: Deploy x64 ------
1>TestApp.vcxproj -> D:\Development\Qt Projects\test\TestApp\x64\Deploy\TestApp.exe
1>windeployqt: "C:/Qt/6.5.3/msvc2019_64/bin\windeployqt.exe" --list target --dir "D:\Development\Qt Projects\test\TestApp\x64\Deploy\\" "D:\Development\Qt Projects\test\TestApp\x64\Deploy\TestApp.exe"
========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
========== Build started at 18:49 and took 02,925 seconds ==========
```

**To Reproduce**
1. Go to ``Extensions`` > ``Qt VS Tools`` > ``Qt Project Settings``
2. Select the ``Qt Deploy Tool`` page.
3. Set the ``Run Deployment Tool`` to ``Yes``
4. Select the arguments you want to set, under the ``windeployqt`` category
5. Build the project

**Desktop (please complete the following information):**
 - Qt VS Tool Version 3.0.1
 - Visual Studio version VS2022
 - Qt 6.5.3 with msvc2019_64

**Comment by HugoX104447** [comment]
Thank you very much for reporting the issue; it has been recorded in our public bug tracker, and all further updates will take place there. Here is the link: [QTVSADDINBUG-1134](https://bugreports.qt.io/browse/QTVSADDINBUG-1134)

**Comment by kaheimri** [comment]
Thank you very much for reporting the issue; it has been recorded in our public bug tracker, and all further updates will take place there. Here is the link: [QTVSADDINBUG-1134](https://bugreports.qt.io/browse/QTVSADDINBUG-1134)

**Comment by kaheimri** [comment]
A fix has been implemented in 8146726 and will be available with the next release.
